sites/default/statement.inc.php

```diff
@@ -403,15 +403,15 @@ function create_HTML_statement($stmt)
     }
```

Comment:
how about at the end of line 394 
`xlt('Chart'); ?></b> <?php echo text($stmt['pid'])`
so the office can easily find the appropriate chart to post the payment
thanks Jerry

Reply:
Okay, but I added my little gravitas :)
@bradymiller Yes, should be xlt() but was going to use a colon and forgot...

Reply:
@sjpadgett , you are forgiven :)